File: 06Adapter/Students/2019/PawelskiBartlomiej/twoWayAdapter/Aircraft.java
package twoWayAdapter;

public class Aircraft implements IAircraft {
    private int height;
    private boolean airbone;

    public Aircraft() {
        height = 0;
        airbone = false;
    }

    @Override
    public boolean airbone() {
        return airbone;
    }

    @Override
    public void takeOff() {
        System.out.println("Engine take out...");
        airbone = true;
        height = 200;
    }

    @Override
    public int height() {
        return height;
    }
}

File: 06Adapter/Students/2019/ZielińskiŁukasz/Aircraft.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Adapter {
    // Target
    public sealed class Aircraft : IAircraft {
        //int height;
        bool airborne;
        public Aircraft() {
            Height = 0;
            airborne = false;
        }
        public void TakeOff() {
            Console.WriteLine("Aircraft engine takeoff");
            airborne = true;
            Height = 200; // Meters
        }
        public bool Airborne
        {
            get { return airborne; }
        }
        public int Height
        {
             get; set; 
        }
    }
}
